ui/tree/src/ops.ts

```diff
@@ -68,7 +68,8 @@ export function countChildrenAndComments(node: Tree.Node) {
 export function merge(n1: Tree.Node, n2: Tree.Node): void {
   if (n2.eval) n1.eval = n2.eval;
   if (n2.glyphs) n1.glyphs = n2.glyphs;
-  n2.comments &&
+  else delete n1.glyphs;
```

Comment:
the function is named `merge`, so it merges the glyphs and comments of `n1` and `n2`.

After this change it only overrides `n1` with `n2`. Which could btw be simplified as simply
```js
n1.glyphs = n2.glyphs
```
instead of
```js
  if (n2.glyphs) n1.glyphs = n2.glyphs;
  else delete n1.glyphs;
```

But that's not the goal.